app/dashboard/management/commands/post_leaderboard_to_slack.py

```diff
@@ -28,28 +28,28 @@
 from marketing.models import LeaderboardRank
 from slackclient import SlackClient
 
-warnings.filterwarnings("ignore", category=DeprecationWarning) 
+warnings.filterwarnings("ignore", category=DeprecationWarning)
 logging.getLogger("requests").setLevel(logging.WARNING)
 logging.getLogger("urllib3").setLevel(logging.WARNING)
 
 
-def post_to_chat(channel, msg):
-    try:
-        from chat.tasks import get_driver
-        chat_driver = get_driver()
+# def post_to_chat(channel, msg):
```

Comment:
How come just commenting these out?

Reply:
yes post_to_chat is just managing the sending to the channel In the past this was sending to slack, probably in the near future will send to discord